charts/pega/README.md

```diff
@@ -480,16 +480,18 @@ Parameter           | Description    | Default value
 `hpa.enableCpuTarget` | Set to true if you want to enable scaling based on CPU utilization or false if you want to disable it | true
 `hpa.enableMemoryTarget` | Set to true if you want to enable scaling based on memory utilization or false if you want to disable it (Pega recommends leaving this disabled) | false
 
-### Setting a Pod Disruption Budget (PDB)
-You can configure a Kubernetes `PodDisruptionBudget` on your tier by setting `pdb.enabled` to `true`.  For more details, see the Kubernetes 
-[Pod Disruption Budgets documentation](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#pod-disruption-budgets).
+### Ensure System Availability during Voluntary Disruptions by Using a Kubernetes Pod Disruption Budget (PDB)
+To safeguard your deployment by limiting the number of Pods running your Pega Platform application that can go down for planned "disruptions", 
+Pega allows you to enable a unique Kubernetes `PodDisruptionBudget` on a tier.  For more details on PDBs, see the Kubernetes [Pod Disruption Budgets documentation](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#pod-disruption-budgets).
 
-The following parameters are mutually exclusive.  You must specify either `pdb.minAvailable` or `pdb.maxUnavailable` but not both.
+You can configure a Kubernetes `PodDisruptionBudget` on your tier by setting `pdb.enabled` to `true` in your values.yaml file.  By default, this value is
+set to `false.`  You must also specify at least one of the following parameters to configure the Pod Disruption Budget.  Keep in mind that these parameters
+are mutually exclusive, and thus only one may be set.  You may provide values that are expressed as percentages (% of pods) or integers (# of pods).  
 
 Parameter             | Description    | Default value
 ---                   | ---       | ---
 `pdb.minAvailable`    | The minimum number of pods in the tier that must remain available at all times  | `1`
-`pdb.maxUnavailable`  | The maximum number of pods in the tier that can be unavailable at one time      | Disabled when `pdb.minAvailable` is specified
+`pdb.maxUnavailable`  | The maximum number of pods in the tier that can be unavailable at one time      | `50%`
```

Comment:
sorry but I think that this is still off...how about?>
The minumum number or percentage of total pods in the tier that can be unavailable at one time; if the number or percentage reaches your minimum, the deployment will not allow you to bring down additional pods until one of the currently unavailable pods is deployed and healthy.

The maximum number or percentage of total pods in the tier that can be unavailable at one time; if the number or percentage reaches your maximum, the deployment will not allow you to bring down additional pods until one of the currently unavailable pods is deployed and healthy.

Reply:
I like the inclusion of percentage, but the suggested description isn't quite technically accurate. Maybe something like this?

> The minimum number or percentage of pods in the tier that must be available. If this minimum is reached, the Kubernetes deployment will not bring down additional pods for voluntary disruptions until more are available and healthy.

> The maximum number or percentage of pods in the tier that can be unavailable. If this maximum is reached, the Kubernetes deployment will not bring down additional pods for voluntary disruptions until more are available and healthy.